derive/src/pyclass.rs

```diff
@@ -420,6 +420,7 @@ pub fn impl_pyimpl(attr: AttributeArgs, item: Item) -> Result<TokenStream2, Diag
 fn generate_class_def(
     ident: &Ident,
     name: &str,
+    tp_name: &str,
```

Comment:
Is there a similar thing to this in Python? A tp_name slot or something? If not, I'd recommend calling this `QUALNAME` or similar.

Reply:
Yes, there is `tp_name`. Here is a test code about it:
```c
.tp_name = "custom.Custom",
```